README.rst

```diff
@@ -104,58 +104,65 @@ List of Modules
 
 The following modules have been completed using a common API:
 
-  * `ALMA Archive <http://astroquery.readthedocs.io/en/latest/alma/alma.html>`_
```

Comment:
I wonder @keflavich whether we should just get rid of this duplicated list in the readme and point to the docs index page instead?

Reply:
yes, I'd be in favor of that.

It is nice to have this information on the README, but the maintenance cost isn't worth it.